socs/agents/hwp_supervisor/agent.py

```diff
@@ -179,9 +192,11 @@ def from_args(cls, args: argparse.Namespace):
             ups_minutes_remaining_thresh=args.ups_minutes_remaining_thresh,
         )
         if args.gripper_iboot_id is not None:
-            self.gripper_iboot = IBootState(args.gripper_iboot_id, args.gripper_iboot_outlets)
+            self.gripper_iboot = IBootState(args.gripper_iboot_id, args.gripper_iboot_outlets,
+                                            args.driver_power_agent_type)
```

Comment:
Here we'll need a new command-line arg `args.gripper_power_agent_type` which is not necessarily the same as the driver power type.